client/components/Coverages/CoverageAddButton/index.tsx

```diff
@@ -47,10 +47,7 @@ export class CoverageAddButton extends React.Component<IProps> {
     onChange(field: string, coverages: Array<DeepPartial<IPlanningCoverageItem>>) {
         this.props.onChange(
             field,
-            [
-                ...(this.props.value ?? []),
-                ...coverages,
-            ],
+            coverages,
```

Comment:
`CoverageAddButton` is used in multiple places, did you check to make sure this change is compatible with other usages?